app/controllers/registration/applicant/SoleTraderIdentificationController.scala

```diff
@@ -49,22 +49,29 @@ class SoleTraderIdentificationController @Inject()(val sessionService: SessionSe
         implicit profile =>
           vatRegistrationService.partyType.flatMap {
             case partyType@(Individual | NETP) =>
-              soleTraderIdentificationService.startSoleTraderJourney(
-                SoleTraderIdJourneyConfig(
+              for {
+                isTransactor <- vatRegistrationService.isTransactor
+                fullNamePageLabel = isTransactor match {
```

Comment:
Generally it's a good practice to handle `boolean` in `if else `rather than pattern matching. A bit more detailed explanation is [here](https://stackoverflow.com/questions/9266822/pattern-matching-vs-if-else) if you are interested.